.pre-commit-config.yaml

```diff
@@ -17,9 +17,9 @@ repos:
       - id: fix-encoding-pragma
         args: ["--remove"]
   - repo: https://github.com/pycqa/flake8
-    rev: 3.7.8
+    rev: 3.8.3
```

Comment:
I'm not actually sure what this setting does, but it smelled like a python version to me.

Reply:
No, this is the version of flake8, but bumping is fine.

I actually don't know if it's pure coincidence or not that all of their versions since 3.7.0 have the same patch numbers as Python.